pkg/clusterfile/clusterfile.go

```diff
@@ -56,13 +56,18 @@ func (c *ClusterFile) GetKubeadmConfig() *kubeadm.KubeadmConfig {
 }
 
 func NewClusterFile(path string) (Interface, error) {
+	clusterFile := new(ClusterFile)
+
+	if path == "" {
```

Comment:
I am wondering if the empty string of path input is valid? 

Is there any cases that empty path is valid? @tgfree7

Reply:
Sealer try to read file from the path. the empty path may cause `ioutil.ReadFile` error.
Was it designed this way?

Reply:
So I think if the path is empty string, you should return an error rather than a nil? WDYT? @tgfree7

Reply:
> So I think if the path is empty string, you should return an error rather than a nil? WDYT? @tgfree7

IMO, `ClusterfileName` is not required for all cases, when sealer run a cluster, if not set the `ClusterfileName`, we can not create a cluster by cmd:
`sealer run kubernetes:v1.19.8 --masters 192.168.0.2,192.168.0.3,192.168.0.4 \
		--nodes 192.168.0.5,192.168.0.6,192.168.0.7 --passwd xxx`

Reply:
Yes, you are right. Let me think which is the better way. For this pull request, I think we can make it move on.